tikv-client/src/main/java/com/pingcap/tikv/TiConfiguration.java

```diff
@@ -125,7 +125,20 @@ public TiConfiguration setTimeoutUnit(TimeUnit timeoutUnit) {
   }
 
   public String getPdAddrsString() {
-    return pdAddrs.toString();
+    return listToString(pdAddrs);
```

Comment:
Maybe you want to cache it? It seems we are using this one as hash key.

Reply:
right